qiskit/quantum_info/states/statevector.py

```diff
@@ -36,7 +36,10 @@ def __init__(self, data, dims=None):
         """Initialize a statevector object.
 
         Args:
-            data (vector_like): a complex statevector.
+            data (vector_like or QuantumCircuit): A complex vector, an ``Operator`` with only one
```

Comment:
You could just explicitly list all the valid types here: np.array, list, Statevector, Operator, QuantumCircuit, Instruction

Reply:
done in 5c59880